src/site/content/en/building-a-split-button-component/index.md

```diff
@@ -0,0 +1,768 @@
+the primary button, it is transparent until otherwise hovered or interacted
+with, and stretched to fill.
+
+{% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/fiSlBJyM8x52b5msQBUY.png",
+alt="ALT_TEXT_HERE", width="800", height="241", class="w-screenshot" %}
+
+```css
+.gui-popup-button {
+  inline-size: 4ch;
+  cursor: pointer;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-inline-start: var(--border);
+  border-start-end-radius: var(--radius);
+  border-end-end-radius: var(--radius);
+}
+```
+
+Layer in hover, focus and active states with [CSS
+Nesting](https://drafts.csswg.org/css-nesting-1/) and the
+[`:is()`](https://web.dev/css-is-and-where/) functional selector:
+
+```css
+.gui-popup-button {
+  …
+
+  &:is(:hover,:focus-within) {
+    background: var(--theme-hover);
+  }
+
+  /* fixes iOS trying to be helpful */
+  &:focus {
+    outline: none;
+  }
+
+  &:active {
+    background: var(--theme-active);
+  }
+}
+```
+
+**These styles are the primary hook for showing and hiding the popup.** When the
+`.gui-popup-button` has `focus` on any of its children, set `opacity`, position
+and `pointer-events`, on the icon and popup.
+
+```css
+.gui-popup-button {
+  …
+  
+  &:focus-within {
+    & > svg {
+      transition-duration: var(--in-speed);
+      transform: rotateZ(.5turn);
+    }
+    & > .gui-popup {
+      transition-duration: var(--in-speed);
+      opacity: 1;
+      transform: translateY(0);
+      pointer-events: auto;
+    }
+  }
+}
+```
+
+With the in and out styles complete, the last piece is to **conditionally
+transition transforms** depending on the user's motion preference:
+
+```css
+.gui-popup-button {
+  …
+
+  @media (--motionOK) {
+    & > svg {
+      transition: transform var(--out-speed) ease;
+    }
+    & > .gui-popup {
+      transform: translateY(5px);
+
+      transition: 
+        opacity var(--out-speed) ease,
+        transform var(--out-speed) ease;
+    }
+  }
+}
+```
+
+A keen eye on the code would notice **opacity is still transitioned** for users
+who prefer reduced motion. 
+
+{% Video 
+  src="video/vS06HQ1YTsbMKSFTIPl2iogUQP73/bsOaOSHe8ZZl0vnZAtmr.mp4",
+  class="w-screenshot",
+  autoplay="true",
+  loop="true",
+  muted="true"
+%}
+
+{% Aside "caution" %}
+Reduced motion doesn't mean no motion.
+{% endAside %}
+
+#### Styling the popup
+
+The `.gui-popup` element is a floating card button list using custom properties
+and relative units to be subtly smaller, interactively matched with the primary
+button, and on brand with its use of color. Notice the icons have less contrast,
+are thinner, and the shadow has a hint of brand blue to it. Like with buttons,
+strong UI/UX is a result of these little details stacking up.
+
+{% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/D7RWfwJW25G3kuzPSnlz.png",
+alt="ALT_TEXT_HERE", width="448", height="422", class="w-screenshot" %}
```

Comment:
```suggestion
{% Img src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/D7RWfwJW25G3kuzPSnlz.png",
alt="A floating card element.", width="448", height="422", class="w-screenshot" %}
```

Reply:
Is this image right? Above you are talking about the icons, this image is just the card.

Reply:
the image is right, but i agree, the flow feels off. I can move the svg styles to after the popup is done, it's def refinement work that's a bit early in the post. i'll make a commit that order this the svg and popup better thanks!